liquidator/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "forta-uma-liquidator-bot",
+  "version": "0.0.1",
+  "description": "Collection of scan agent handlers that monitor UMA",
```

Comment:
Agent that monitors UMA protocol for liquidation opportunities.

Reply:
done